docs/governance/governance.md

```diff
@@ -0,0 +1,238 @@
+
+As with all decisions, the decision to remove a Team member is subject to article 3.4.
+
+## 6 Steering committee
+
+### 6.1 Responsibilities
+
+The Steering Committee shall be responsible for:
+
+- Outlining the technical direction of the EESSI project and setting priorities
+- Conflict resolution: if there is a conflict between the people with the aforementioned roles, the steering committee shall mediate the dispute and make a final decision if consensus cannot be reached
+- The [EESSI Charter](charter.md)
+- The [EESSI Governance](governance.md)
+- The [EESSI Policies](policies.md)
+
+### 6.2 Members and Chairs
+
+The Steering Committee is made up of community members with sustained and recognized contributions over time. Members take part in the Steering Committee as private individuals (i.e. they do not represent their employer(s)).
+
+#### 6.2.1 Termination of membership
+
+Membership of the Steering Committee can terminate in three ways:
+
+- A Member can resign
+- A Member may be voted out. In this case, the vote needs to be unanimous between the other Members. This procedure is intended to provide a mechanism for addressing extraordinary circumstances where a members behavior or actions are deemed severely detrimental to the committee's function or reputation, and/or to the EESSI project.
+- If a Member is inactive for 6 months and does not reply to communication from the rest of the Steering Committee, the Member may be voted out by the other Steering Committee members. In this case, [regular voting rules](#voting-by-sc) apply.
+
+#### 6.2.2 New members
+
+New Members must be recommended by one or more Steering Committee members.
+
+The Steering Committee will make the recommendation known to the community, and collect feedback on the recommendation.
+The Steering Committee then weighs the feedback, and votes on whether to accept the new member.
+
+In the selection process, the Steering Committee will consider the following:
+
+- The Steering Committee ideally consists of an odd number of Members;
+- No more than 30% of the Members should be employed by commercial entities;
+- The number of Members working for the same company should be limited to 1;
+- The number of Members from the same country should be limited to 2;
+- The composition of the Steering Committee should reflect the EESSI community;
+
+Note that if Members change jobs, or move to another country, some of the above criteria that were taken into account during selection may no longer be satisfied. It is up to the Steering Committee to decide if this is problematic, and if so, try to restore balance by requesting someone to resign.
+
+All new Members will appoint an alternate that may represent them and vote on their behalf in case they are unavailable. If a Member resigns or is voted out from the Steering Committee, their alternate immediately loses any rights they had as an alternate.
```

Comment:
Maybe we should make this optional, and even leave the door open for temporary alternate (for 1 meeting)?